### PR TITLE
Integration tests for UserController

### PR DESCRIPTION
**Pull request by KacFlor**

### CODE REVIEW

File: src/test/java/com/KacFlor/ShopSpring/controller/UserControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    @Test
+    public void testDeleteUserById(){
+
+        Integer userId = 1;
+        given(userService.deleteUserById(userId)).willReturn(true);
+
+        boolean result = userController.deleteById(userId);
+
+        assertEquals(result, true);
```
Comment (Ficzer) [inline src/test/java/com/KacFlor/ShopSpring/controller/UserControllerTest.java]: Ok but wasn't it supposed to test HTTP responses, that is the point of this test

Comment (Ficzer) [reply src/test/java/com/KacFlor/ShopSpring/controller/UserControllerTest.java]: Also check proper security token

Comment (KacFlor) [reply src/test/java/com/KacFlor/ShopSpring/controller/UserControllerTest.java]: done

File: src/test/java/com/KacFlor/ShopSpring/controller/UserControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+        List<User> userList = Arrays.asList(
+                new User("Test1", "password1", new Customer(), Role.USER),
+                new User("Test2", "password2", new Customer(), Role.USER),
+                new User("Test3", "password3", new Customer(), Role.USER)
+        );
+        when(userService.getAllUsers()).thenReturn(userList);
+
+        mockMvc.perform(get("/user/users"))
+                .andExpect(status().isOk());
```
Comment (Ficzer) [inline src/test/java/com/KacFlor/ShopSpring/controller/UserControllerTest.java]: Check also content of response

Comment (KacFlor) [reply src/test/java/com/KacFlor/ShopSpring/controller/UserControllerTest.java]: done

**Review by Ficzer — APPROVED** [review]

**Pull request merged**